web/wp-config.php

```diff
@@ -14,180 +14,75 @@
  */
 require_once( $rootPath . '/vendor/autoload.php' );
 
-/*
- * Fetch .env
- */
-if ( ! isset( $_ENV['PANTHEON_ENVIRONMENT'] ) && file_exists( $rootPath . '/.env' ) ) {
-	$dotenv = Dotenv\Dotenv::create($rootPath);
-	$dotenv->load();
-	$dotenv->required( array(
-		'DB_NAME',
-		'DB_USER',
-		'DB_HOST',
-	) )->notEmpty();
-}
-
-/**
- * Disallow on server file edits
- */
-define( 'DISALLOW_FILE_EDIT', true );
-define( 'DISALLOW_FILE_MODS', true );
-
 /**
  * Force SSL
  */
 define( 'FORCE_SSL_ADMIN', true );
```

Comment:
I guess you picked this up from pantheon-systems/example-wordpress-composer. We probably don't want to add it to pantheon-systems/wordpress to avoid conflicts, so this will be another difference.

Reply:
Could we move this to wp-config-pantheon.php, and guard it to only be defined if it is not already defined?

Reply:
I guess you already did this?